tests/libyui/menu_hotkeys_1177760_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative "rspec_tmux_tui"
+
+describe "Menu Item" do
+  bug = "1177760"
+  around(:each) do |ex|
+    @base = "menu_hotkeys_#{bug}"
+
+    yast_ncurses = "#{__dir__}/yast_ncurses"
+    example_dir = "/usr/share/doc/packages/yast2-ycp-ui-bindings/examples"
+    @tui = TmuxTui.new_session "#{yast_ncurses} #{example_dir}/MenuBar1.rb"
+
+    ex.run
+
+    if @tui.has_session?
+      @tui.kill_session
```

Comment:
why two methods? I would have just one like `kill_session` that internally check if session exists.

Reply:
NP: Maybe we should print a warning on STDERR in this case, each test tries to finish cleanly (by pressing "Quit" or "Close" button), if the test is still running then something went wrong.